web/app.go

```diff
@@ -143,6 +148,7 @@ func NewAppWithDeps(config *Config, deps Dependencies) (*App, error) {
 	webEngine.GET("/", HomeHandler)
 	webEngine.GET("/about", NewAboutHandler(deps.subscriptionsService))
 	webEngine.GET("/hosts", NewHostListHandler(deps.consul, deps.tagsService))
+	webEngine.GET("/hosts-next", NewHostListNextHandler(deps.hostsNextService))
```

Comment:
Have you considered attaching always to the same route, but changing handler depending on a quickly baked feature flag (e.g. an environment variable or an internal viper setting)?

Reply:
@stefanotorresi we could but I need both routes for development and also this is gonna go away pretty soon...

Reply:
All righty then, carry on!